workers/repositoryactioncounter.py

```diff
@@ -34,8 +34,16 @@ def batch_query():
         if min_id is None or max_id is None:
             return
 
-        # 4^log10(total) gives us a scalable batch size into the billions.
-        batch_size = int(4 ** log10(max(10, max_id - min_id)))
+        # Check for the number RAC entries vs number of repos. If they are the same,
+        # nothing more to do.
+        repo_count = model.repository.get_repository_count()
+        rac_count = model.repositoryactioncount.found_entry_count(yesterday)
+        if rac_count >= repo_count:
```

Comment:
`==` or `>=`? Is there a case where the RAC should ever be higher than the repo count (repo deletions?)

Reply:
Repository gets deleted and removed between one check and the other.